package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@ukhomeoffice/cop-react-form-renderer",
-  "version": "2.1.0",
+  "version": "2.2.0",
```

Since this is an incomplete feature, it should go down as a pre-release version until such time as it is complete.
```suggestion
  "version": "2.2.0-alpha",
```